packages/grafana-ui/src/components/PanelChrome/PanelContext.ts

```diff
@@ -1,5 +1,6 @@
 import { EventBusSrv, EventBus } from '@grafana/data';
 import React from 'react';
+import { GraphNGLegendEventMode } from '../GraphNG/types';
```

Comment:
Can we move this type to PanelChrome?

Reply:
Ah yes, I was gonna get around to that but forgot :)

Reply:
And I think we can also rename it to sth GraphNG agnostic. `SeriesToggleMode`?